pkg/runner/cluster_k8s.go

```diff
@@ -243,7 +243,7 @@ func (*ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow io.Wri
 		if job.Status.Active == 0 && (job.Status.Succeeded > 0 || job.Status.Failed > 0) {
 			break
 		}
-		if time.Since(start) > 5*time.Minute {
+		if time.Since(start) > 10*time.Minute {
```

Comment:
Ideally, we wouldn't have hard-coded timeouts like this. But we can tackle that later.